### PR TITLE
Add some sleep steps into shutdown

### PR DESCRIPTION
**Pull request by Frank-GER**
Adding some short sleep after several steps of the shutdown procedure.
This should slow it down enough that threads not unintentionally kill each other,
which we have seen especially on windows.

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -238,10 +238,13 @@ void Shutdown(NodeContext& node)
     StopRPC();
     StopHTTPServer();
     // SYSCOIN
+    // Adding sleep after several steps to aviod occational problems on windows
```
Comment (sidhujag) [inline src/init.cpp]: fix spelling

Comment (Frank-GER) [reply src/init.cpp]: fixed

File: src/init.cpp [hunk 2]
```diff
@@ -238,7 +238,7 @@ void Shutdown(NodeContext& node)
     StopRPC();
     StopHTTPServer();
     // SYSCOIN
-    // Adding sleep after several steps to aviod occational problems on windows
+    // Adding sleep after several steps to avoid occational problems on windows
```
Comment (sidhujag) [inline src/init.cpp]: still didnt fix "occational"

Comment (Frank-GER) [reply src/init.cpp]: fixed too

**Review by sidhujag — APPROVED** [review]

**Pull request merged**